middle_end/flambda2/to_cmm/to_cmm_set_of_closures.ml

```diff
@@ -280,6 +280,8 @@ end = struct
         else if starting_offset = slot_offset
         then acc
         else
+          (* The space between slot offsets has to be padded with tagged 0, as
+             it is scanned by the GC during compaction. *)
```

Comment:
I think we should specify here that the constraint is not only that the value put here has to be scannable, but that it has to be a very specific value, and add a link/comment to the relevant part of the compaction C code.